MekHQ/src/mekhq/campaign/personnel/SpecialAbility.java

```diff
@@ -551,9 +551,10 @@ public static boolean isWeaponEligibleForSPA(EquipmentType et, PersonnelRole rol
             return false;
         }
 
+        // Should only apply to Large Ship-mounted weapons
         if (wt.getAtClass() == WeaponType.CLASS_NONE ||
                 wt.getAtClass() == WeaponType.CLASS_POINT_DEFENSE ||
-                wt.getAtClass() >= WeaponType.CLASS_CAPITAL_LASER) {
+                (wt.getAtClass() >= WeaponType.CLASS_CAPITAL_LASER && wt.getAtClass() <= WeaponType.CLASS_TELE_MISSILE)) {
```

Comment:
<= ?

Reply:
> <= ?

Yep.  That span of AT Class definitions all deal with Capital / Large Ship stuff; after `CLASS_TELE_MISSILE` there are a couple of newer classes that shouldn't be excluded anymore.